src/utils/diff/parsers/golang/index.ts

```diff
@@ -1,13 +1,20 @@
 import { BaseParser } from '../common'
 
+const authCapturePattern = /\w*/
```

Comment:
minor thing but I've been using the word "Capture" in the variable name to denote that its meant to capture something with a capture group, vs. a regular "pattern"

Reply:
True, do you think all the others should be updated that have non-capturing groups?

Reply:
depends how useful you find that distinction in the variable names..it seems like it might be a bit of effort to change for not much gain